site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb

```diff
@@ -0,0 +1,1159 @@
+    "\n",
+    "<table>\n",
+    "  <tr><td>\n",
+    "    <img src=\"https://www.tensorflow.org/images/custom_estimators/full_network.png\"\n",
+    "         alt=\"A diagram of the network architecture: Inputs, 2 hidden layers, and outputs\">\n",
+    "  </td></tr>\n",
+    "  <tr><td align=\"center\">\n",
+    "    <b>그림 2.</b> A neural network with features, hidden layers, and predictions.<br/>&nbsp;\n",
+    "  </td></tr>\n",
+    "</table>\n",
+    "\n",
+    "그림 2의 모델이 훈련된 다음 레이블 되어있지 않은 데이터를 제공했을때, 모델은 주어진 데이터의 3가지(주어진 레이블의 개수) 예측을 출력합니다. 이러한 예측은 *[추론(inference)](https://developers.google.com/machine-learning/crash-course/glossary#inference)* 이라고 불립니다. 이 샘플에서 출력의 합은 1.0입니다. 그림 2에서 예측은 *Iris setosa* `0.02`, *Iris versicolor* `0.95`, *Iris virginica*에 `0.03`로 주어집니다. 이는 모델이 95%의 확률로 주어진 데이터를 *Iris versicolor*로 예측한다는 것을 의미합니다. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "W23DIMVPQEBt"
+   },
+   "source": [
+    "### 케라스를 사용한 모델 생성\n",
+    "\n",
+    "텐서플로의 [tf.keras](https://www.tensorflow.org/api_docs/python/tf/keras) API는 모델과 층을 생성하기 위한 풍부한 라이브러리를 제공합니다. 케라스가 구성 요소를 연결하기 위한 복잡함을 모두 처리해 주기 때문에 모델을 구축하고 실험하는 것이 쉽습니다.\n",
+    "\n",
+    "[tf.keras.Sequential](https://www.tensorflow.org/api_docs/python/tf/keras/Sequential)은 여러 층을 연이어 쌓은 모델입니다. 이 구조는 층의 인스턴스를 취하며, 아래의 경우 각 층당 10개의 노드(node)를 가지는 2개의 [완전 연결((Dense) 층](https://www.tensorflow.org/api_docs/python/tf/keras/layers/Dense)과 3개의 예측(레이블의 수) 노드를 가지는 출력 층으로 구성되어있습니다. 첫 번째 층의 `input_shape` 매개변수는 데이터셋의 특성의 수와 관계있습니다."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "2fZ6oL2ig3ZK"
+   },
+   "outputs": [],
+   "source": [
+    "model = tf.keras.Sequential([\n",
+    "  tf.keras.layers.Dense(10, activation=tf.nn.relu, input_shape=(4,)),  # 입력의 형태가 필요합니다.\n",
+    "  tf.keras.layers.Dense(10, activation=tf.nn.relu),\n",
+    "  tf.keras.layers.Dense(3)\n",
+    "])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "FHcbEzMpxbHL"
+   },
+   "source": [
+    "*[사용가능한 활성화 함수(activation function)](https://developers.google.com/machine-learning/crash-course/glossary#activation_function)*는 각 층에서 출력의 크기를 결정합니다. 이러한 비선형성은 중요하며, 활성화 함수가 없는 모델은 하나의 층과 동일하다고 생각할 수 있습니다. 사용 가능한 [활성화 함수](https://www.tensorflow.org/api_docs/python/tf/keras/activations)는 많지만, [ReLU](https://developers.google.com/machine-learning/crash-course/glossary#ReLU)가 은닉층에 주로 사용됩니다. \n",
```

Comment:
Shape is often translated '크기'. But I think 'shape' in this sentence means '형태'. Because activation func. doesn't change Tensor shape. And just remove '사용가능한'.
I suggest '사용가능한 활성화 함수(activation function)는 각 층에서 출력의 크기를 결정합니다' --> '활성화 함수(activation function)는 각 층에서 출력의 형태를 결정합니다'